scripts/analyze.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Use the same bash in every script